include/roboteam_ai/world_new/Robot.hpp

```diff
@@ -0,0 +1,178 @@
+//
+// Created by john on 12/16/19.
+//
+
+#ifndef RTT_ROBOT_HPP
+#define RTT_ROBOT_HPP
+
+#include <roboteam_proto/RobotFeedback.pb.h>
+#include "roboteam_utils/Angle.h"
+
+#include "roboteam_proto/WorldRobot.pb.h"
+
+
+#include "control/shotControllers/ShotController.h"
+#include "control/numTrees/NumTreePosControl.h"
+#include "control/BasicPosControl.h"
+#include "control/ballHandling/BallHandlePosControl.h"
+#include "world_new/Team.hpp"
+
+
+namespace rtt::world_new::robot {
+
+    /**
+     * Geneva driver gone
+     * 30 / 50 watt motor flag
+     *
+     * robot still changes:
+     *  battery
+     *  spinner speed
+     */
+    class Robot {
+    private:
+        uint32_t id;
+        Team team;
+        uint8_t wattage = 30;
```

Comment:
wat does this variable mean?

Reply:
I figure this is the power of the motors since Jesse wanted to seperate between 30 and 50W motors and be able to run both. It's probably clearer to give robots an enum called Generation or robotType or so.

Reply:
Will change this today when at the office.